### DISCUSSION TITLE
Segment Anything 2 released

### DISCUSSION THREAD

**Discussion by brianmcdonald** [opengeos/segment-geospatial#301] [Q&A] [answered]
See https://ai.meta.com/blog/segment-anything-2/ and https://github.com/facebookresearch/segment-anything-2

Meta just released Segment Anything 2. I wonder what improvements it has for geospatial segmentation and if SAM2 can be incorporated  into Segment Geospatial.

**Comment by giswqs** [comment, marked as answer]
See https://github.com/opengeos/segment-geospatial/issues/300
